### DISCUSSION TITLE
Please Provide Config Files

### DISCUSSION THREAD

**Discussion by lemmingDev** [moonglow/FlashForge_Marlin#74] [Ideas]
Hi

Hoping you would consider providing the config files used to create the pre-compiles hex files in your releases.

I realise that you should simply define the printer type, UI desired, extruder swap and pully color you have at the top of the configuration.h file, but with other options such, reversing extruder stepper motors and linear advance etc, it would be good to have them as a base for those of us who want to do modifications.

Eg, The FlashForge Dreamer is set to 140mm max Z height, however mine is able to print to 160mm, so I'd like to change that, along with perhaps adding BL Touch etc. I know how to add/edit those features in myself, but it would be good to have a much more narrowed down starting point than the generic one provided

Even better I guess, and to keep just the one configuration file, defines could be added for linear advance and the extruder direction fix. Basically all of the options your pre-compiled have

Thanks for your consideration

**Comment by moonglow** [comment]
Hi @lemmingDev 

>Hoping you would consider providing the config files used to create the pre-compiles hex files in your releases.

My repo is already has all config files what you need. About my precompiled version, i use command line calls to platformio to build it. Please check my `marlin_builder.sh` it shows you all option and steps for build firmware. No other private configs or files was used.

>Even better I guess, and to keep just the one configuration file

All my settings located only in one configuration file, you just choose what you need to enable or disable ( In my case i do it via build script ).

**Comment by lemmingDev** [comment]
Ah - ok

Thanks for the clarification and getting back to me

I'll take a look at that file and sus it out from there

Much appreciated
